docs/api-reference/layers/bitmap-layer.md

```diff
@@ -153,6 +153,28 @@ The color to use for transparent pixels, in `[r, g, b, a]`. Each component is in
 
 The color to tint the bitmap by, in `[r, g, b]`. Each component is in the `[0, 255]` range.
 
+### Callbacks
+
+From v8.4, `BitmapLayer` provides information on which pixel was picked. Picking callbacks (`onHover`, `onClick`) now contain an additional field `bitmap` field, with `pixel` coordinates.
+
+```typescript
+{
+  PickingInfo,
```

Comment:
I don't know what this means. `bitmap` is supposed to be part of the pickingInfo object?